src/main/kotlin/org/wa/rceditor/domain/RCManager.kt

```diff
@@ -0,0 +1,11 @@
+package org.wa.rceditor.domain
+
+import io.reactivex.Completable
+import io.reactivex.Maybe
+import org.wycliffeassociates.resourcecontainer.ResourceContainer
+
+interface RCManager {
```

Comment:
Like mentioned above, probably should make this name more descriptive to help clarify what "manage" means in this context